src/index.js

```diff
@@ -1,33 +1,18 @@
 // css files here
 import './style.css';
+import dummy from './modules/dummy.js';
```

Comment:
- [ ] please it would be better if you remove the unused code in order to keep the file clean and pro

Reply:
ok thanks for helpful advice  I will  remove it